common/socks5/requests/src/response.rs

```diff
@@ -175,6 +189,17 @@ impl Socks5ResponseContent {
                     .chain(res.into_bytes().into_iter())
                     .collect()
             }
+            Socks5ResponseContent::Query(query) => {
+                use bincode::Options;
+                let query_bytes: Vec<u8> = bincode::DefaultOptions::new()
```

Comment:
it seems we're creating that `bincode::DefaultOptions::new().with_big_endian().with_varint_encoding()` in multiple places. maybe we should extract it to a separate `make_bincode_serializer` or `make_queries_serializer` function so it'd be easier to update it (if we ever decided to)